runelite-client/src/main/java/net/runelite/client/plugins/skybox/Skybox.java

```diff
@@ -352,14 +352,17 @@ private int chunkData(int cx, int cy, int plane, ChunkMapper chunkMapper)
 		}
 
 		int cv = chunks[(stride * (cy - y1)) + (cx - x1)];
+
```

Comment:
Now that there are no more code changes to Skybox.java, remove the added newlines in this file.

Reply:
Yep, will do